src/transacoes/Transacao.test.js

```diff
@@ -3,19 +3,19 @@ import { render, screen } from '@testing-library/react';
 import Transação from './Transacao';
 
 test('mostra estabelecimento', () => {
```

Comment:
Acho importante aqui nos testes deixar um pouco mais descritivo, pode usar mais describe e it também sem problemas, mesmo que o código fique um pouco maior, porém mais descritivo.